public/src/components/lab/LabResult.tsx

```diff
@@ -35,8 +35,8 @@ export class LabResult extends React.Component<ILabResult, ILabResultState> {
             approveButton = <div className="btn lab-btn approve-btn"> <button type="button"
                 id="approve"
                 className={this.setButtonColor("approve")}
```

Comment:
I would probably move the <button... to a new line above.

Reply:
Done.